src/Marten.Storyteller/Marten.Storyteller.csproj

```diff
@@ -1,25 +1,24 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
-    <AssemblyName>Marten.Storyteller</AssemblyName>
-    <OutputType>Exe</OutputType>
-    <PackageId>Marten.Storyteller</PackageId>
-    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
-    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
-    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
-  </PropertyGroup>
+    <PropertyGroup>
+        <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
+        <AssemblyName>Marten.Storyteller</AssemblyName>
+        <OutputType>Exe</OutputType>
+        <PackageId>Marten.Storyteller</PackageId>
+        <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+        <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+        <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    </PropertyGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\Marten.Testing\Marten.Testing.csproj" />
-  </ItemGroup>
+    <ItemGroup>
+        <ProjectReference Include="..\Marten.Testing\Marten.Testing.csproj" />
+    </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Storyteller" Version="5.3.5" />
-  </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="Storyteller" Version="5.4.0" />
+    </ItemGroup>
 
-  <ItemGroup>
-    <DotNetCliToolReference Include="dotnet-storyteller" Version="5.2.0" />
-  </ItemGroup>
-
-</Project>
+    <ItemGroup>
+        <DotNetCliToolReference Include="dotnet-storyteller" Version="5.2.0" />
```

Comment:
Please update `dotnet-storyteller` to version `5.4.0`

Reply:
Fixed 👍